<launch>
	
	<arg name="imu_topic" default="/mavros/imu/data" />
	<arg name="position_topic" default="/eaglered/position"/>
	
    <node name="msf_pose_sensor" pkg="msf_updates" type="pose_sensor" clear_params="true" output="screen">
          <remap from="msf_core/imu_state_input" to="$(arg imu_topic)" />a
          <remap from="msf_updates/transform_input" to="$(arg position_topic)" />
          
          <rosparam file="$(find msf_updates)/pose_sensor_fix_erl.yaml"/>
    </node>
</launch>